introduction/from_flat_php_to_symfony2.rst

```diff
@@ -370,7 +370,7 @@ on the requested URI::
         echo '<html><body><h1>Page Not Found</h1></body></html>';
     }
 
-For organization, both controllers (formerly ``index.php`` and ``show.php``)
+For organization, both controllers (formerly ``/index.php`` and ``/index.php/show``)
```

Comment:
The problem is that the article talks about the controllers related to the list_action() and show_action() functions. Given that this controllers don't have a name, or a function or anything to refer to, I prefer to mention the original URI of each controller (/index.php and /index.php/show) to avoid issues like the original bug report. Otherwise, how can we solve this? Thanks!